api/reports.py

```diff
@@ -5,59 +5,76 @@
 from pydantic import BaseModel
 from typing import List, Optional
 from libs.utils.reports import ReportsService
-from pathlib import Path
 
 reports_api = FastAPI()
 
 reports = ReportsService()
 
+
 class HourlyReport(BaseModel):
     hours: List[int]
     detected_objects: List[float]
     violating_objects: List[float]
+    detected_faces: List[float]
+    faces_with_mask: List[float]
+
     class Config:
         schema_extra = {
             'example': [{
                 'hours': list(range(0, 23)),
                 'detected_objects': [0., 7273.5, 10011.9, 0.5, 0., 7273., 10011., 0., 0., 7273., 10011., 0.,
                                      0., 7273.1, 10011.2, 0.5, 0., 7273., 10011., 0., 0., 7273., 10011., 0.],
                 'violating_objects': [0., 4920.5, 6701.3, 0.5, 0., 4920., 6701., 0., 0., 4920., 6701., 0.,
-                                      0., 4920.1, 6701.44, 0.1, 0., 4920., 6701., 0., 0., 4920., 6701., 0.]
+                                      0., 4920.1, 6701.44, 0.1, 0., 4920., 6701., 0., 0., 4920., 6701., 0.],
+                'detected_faces': [0., 7273.5, 10011.9, 0.5, 0., 7273., 10011., 0., 0., 7273., 10011., 0.,
+                                   0., 7273.1, 10011.2, 0.5, 0., 7273., 10011., 0., 0., 7273., 10011., 0.],
+                'faces_with_mask': [0., 4920.5, 6701.3, 0.5, 0., 4920., 6701., 0., 0., 4920., 6701., 0.,
+                                    0., 4920.1, 6701.44, 0.1, 0., 4920., 6701., 0., 0., 4920., 6701., 0.]
             }]
         }
 
 
-class DailyReport(BaseModel):
-    dates: List[str]
+class IntReport(BaseModel):
     detected_objects: List[int]
     violating_objects: List[int]
+    detected_faces: List[int]
+    faces_with_mask: List[int]
```

Comment:
We should maybe just change the original to an int  report, right?

Reply:
The hourly report returns float instead of int. For that reason in created a IntReport instead of a generic Report.

If we return the same type in all the reports, we can create a single class called `Report` and inherit always from there.
If you agree, we can implement all the reports returning floats or change the hourly report to int.

Reply:
I'd just change the floats to ints and ask the frontenders if anything breaks (doubt so).

We had talked about doing so and I think it makes sense

Reply:
Changed. I removed all the float indicators from the reports.